lib/matplotlib/axes/_axes.py

```diff
@@ -5433,7 +5441,7 @@ def imshow(self, X, cmap=None, norm=None, aspect=None,
             The filter radius for filters that have a radius parameter, i.e.
             when interpolation is one of: 'sinc', 'lanczos' or 'blackman'.
 
-        resample : bool, optional
+        resample : bool, default: False
```

Comment:
The default for `AxesImage` is False; the default for imshow is `None`, which means the default here should be the `'image.resample'` rcParam.